frontend/compositions/header/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react"
+import NPDCLogo from './assets/NPDCLogo.svg'
+import styles from './header.module.css'
+
+
+export default function Header() {
+  let [selected, setSelected] = useState('Search');
+
+  const handleNavChange = (e: any) => {
+    let value = e.target.innerText;
+    setSelected(value);
+  }
+
+  return (
+    <header className={styles.wrapper}>
```

Comment:
If you like, something I've found helpful is to deconstruct the `classNames` from `styles` so that you don't have to write `styles.` in each of them!

Total non-blocker tho!

Reply:
@srsexton94 this is super helpful - thank you for suggestion!